packages/Uploader/src/components/File/File.js

```diff
@@ -70,12 +69,14 @@ function File(props) {
         return (
           <Popover isDark isEager>
             <Popover.Tip />
-            <Popover.Trigger>
+            <Popover.Trigger tabIndex={-1}>
```

Comment:
why set this to -1? how can I cancel it by keyboard then?

Reply:
Ahh, yes you're right. It is able to cancel by keyboard currently, but it should actually be implemented by passing in a function to render the button. Then we won't need the tabIndex=-1. I will change this!